changelog/fragments/k8s-1.24-bump.yaml

```diff
@@ -0,0 +1,63 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+
+# TODO(everettraven): update changelog
+entries:
+  - description: >
+      Bump dependencies to support Kubernetes 1.24. 
+      Projects scaffolded with the go/v3 plugin now support Go 1.18 - [More info](https://github.com/kubernetes-sigs/kubebuilder/commit/02da15bfc32c9deb030a811d16da45ace5cc8db8).
+      
+      Dependency bumps that occurred are as follows:
+      
+      - github.com/operator-framework/helm-operator-plugins --> commit d3967d2ae2acf40cca1bf4215508b10a55ea6755
+      - github.com/operator-framework/java-operator-plugins --> v0.5.1
+      - helm.sh/helm/v3 --> v3.9.0
+      - k8s.io/api --> v0.24.0
+      - k8s.io/apiextensions-apiserver --> v0.24.0
+      - k8s.io/apimachinery --> v0.24.0
+      - k8s.io/cli-runtime --> v0.24.0
+      - k8s.io/client-go --> v0.24.0
+      - k8s.io/kubectl --> v0.24.0
+      - sigs.k8s.io/controller-runtime --> v0.12.1
+      - sigs.k8s.io/controller-tools --> v0.9.0
+      - sigs.k8s.io/kubebuilder/v3 --> commit eea565cb3f508d7bcf10831d2b761dbf59af6db0
+
+      Additional Changes:
+
+      - :bug: fix channel naming validation by @camilamacedo86 [More info](https://github.com/operator-framework/api/pull/231)
+      - Add CRD description check validation by @ryantking [More info](https://github.com/operator-framework/api/pull/234)
+      - :sparkles: (GoodPracticesValidator) : Add a new check to warn authors when permissions to create CRDs are found by @camilamacedo86 [More info](https://github.com/operator-framework/api/pull/241)
+
```

Comment:
@everettraven, could you please replace it by:

```
# entries is a list of entries to include in
# release notes and/or the migration guide
entries:
  - description: >
      For operator-sdk bundle validate (`--select-optional name=good-practices`), fix channel naming validation [More info](https://github.com/operator-framework/api/pull/231)
    kind: "bugfix"
  - description: >
      For operator-sdk bundle validate `--select-optional name=good-practices`), add new check to ensure that CRD descriptions are not empty [More info](https://github.com/operator-framework/api/pull/234)
    kind: "addition"
  - description: >
      For operator-sdk bundle validate `--select-optional name=good-practices`), add a new check to warn authors when permissions to create CRDs are found [More info](https://github.com/operator-framework/api/pull/241)
```

Reply:
Addressed in other comments